### PR TITLE
Get organization users from user-manager

### PR DESCRIPTION
**Pull request by Ladicle**

### CODE REVIEW

File: deployments/rbac-server/values.yaml [hunk 1]
```diff
@@ -8,22 +8,24 @@ apiKeyCache:
   syncInterval: 10s
   userManagerServerInternalAddr: llm-operator-user-manager-server-internal-grpc:8082
 
+roleScopesMap:
+  owner:
+  - api.model.read
```
Comment (kkaneda) [inline deployments/rbac-server/values.yaml:13]: Should we have a separate role for inference? Currently we use `api.model`, but I'm wondering if it should be `api.chat.completions`. 

https://github.com/llm-operator/inference-manager/blob/main/server/internal/server/server.go#L75

**Review by kkaneda — APPROVED** [review]
LG!

File: deployments/rbac-server/values.yaml (hunk 1 above)
Comment (Ladicle) [reply deployments/rbac-server/values.yaml:13]: Or `model.request` if we follow the openai.
Here is the openai's scope spec:
```
{
    "name": "Models",
    "description": "List models this organization has access to",
    "permissions_to_scopes": {
      "read": [
        "model.read",
        "api.model.read"
      ]
    },
    "endpoints": [
      "/v1/models"
    ]
},
{
    "name": "Model capabilities",
    "description": "Create chat completions, audio, embeddings, and images",
    "permissions_to_scopes": {
      "write": [
        "model.request",
        "api.model.request"
      ]
    },
    "endpoints": [
      "/v1/audio",
      "/v1/chat/completions",
      "/v1/embeddings",
      "/v1/images",
      "/v1/moderations"
    ]
  },
```

Comment (kenji-cloudnatix) [reply deployments/rbac-server/values.yaml:13]: I see. Sounds good!

**Pull request merged**